contrib/ansible/roles/kubernetes-addons/templates/kube-addons.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
```

Comment:
Also, can you help me understand why this needs to be run on every boot?  Shouldn't it only need to be run once? Why is this a systemd service?